headers/types/common/common.h

```diff
@@ -992,6 +992,145 @@ struct wavi_data {
 };
 ASSERT_SIZE(struct wavi_data, 20); // Likely longer
 
+// Specifies an entry function for a thread's execution
```

Comment:
FYI I rephrased the comment a bit so it couldn't be mistaken for being a "pre-startup" routine that might return quickly (similar to the exit function). This way it's more obvious that this is the thread's "main function". Lmk if that doesn't make sense.